litmus/PTX/Proxy-MP-gen-sur-surfaceFence1.litmus

```diff
@@ -1,5 +1,6 @@
-PTX MPcta
-"Two thread example with a constant proxy fence"
+PTX Proxy-MP-gen-sur-surfaceFence1
+"MP among generic and surface operations, MP is guaranteed because surface fence is along the base causality path from weak st to suld."
+"surface proxy could be in either of the threads"
```

Comment:
I think you are right the behavior is not allowed because the store is GEN, the load is tagged as surface and there is a surface fence in between. But does it make sense to have a surface load to a variable that is defined as texture? (I asked Daniel the same in an issue)

Reply:
I think I agree that it doesn't make sense to have the combination of suld and a texture location, and I feel it should forbidden in practice as Daniel said. The t is updated to s now.